lab_4/likelihood_text_generator_score_eight_test.py

```diff
@@ -157,7 +157,7 @@ def test_generate_next_word_incorrect_context(self):
 
         trie = NGramTrie(3, encoded)
 
-        bad_inputs = [[], {}, (2000, 1000, ), None, 9, 9.34, True]
+        bad_inputs = ([], {}, tuple(), (2000, ), None, 9, 9.34, True)
```

Comment:
and why not list, but tuple?

Reply:
for consistency of a whole project, including main.py